tests/test_archived_contacts_support.py

```diff
@@ -0,0 +1,59 @@
+import tap_tester.menagerie as menagerie
+import tap_tester.runner as runner
+import tap_tester.connections as connections
+
+from base import XeroScenarioBase, preserve_refresh_token
+
+class TestArchivedContacts(XeroScenarioBase):
+
+    def setUp(self):
+        self.includeArchivedContacts = None
+        super().setUp()
+
+    def name(self):
+        return "tap_tester_xero_common_connection"
+
+    def get_properties(self):
+        properties = super().get_properties()
+
+        # includeArchivedContacts is an optional property for configuration
+        if self.includeArchivedContacts:
+            properties["includeArchivedContacts"] = self.includeArchivedContacts
+
+        return properties
+
+    def get_records_from_xero_platform(self):
+        only_active_contacts = runner.run_check_job_and_check_status(self)
+
+        found_catalogs = menagerie.get_catalogs(self.conn_id)
+        self.check_all_streams_in_catalogs(found_catalogs)
+        # Replicating the contacts stream
+        self.select_specific_catalog(found_catalogs, "contacts")
+
+        # clear state and run the actual sync
+        menagerie.set_state(self.conn_id, {})
+        runner.run_sync_job_and_check_status(self)
+        records = runner.get_upserts_from_target_output()
+
+        return records
+
+
+    def test_get_archived_contacts(self):
```

Comment:
These changes look great. One more thing though: could you also assert the number of records is greater in the second sync than the first? That would give more confidence that the api and tap behavior match our expectations.

Reply:
@kspeer825 Incorporated the change you suggested above